modules/bloop-rifle/src/main/scala/scala/build/blooprifle/Version.scala

```diff
@@ -0,0 +1,7 @@
+package scala.build.blooprifle
+import org.apache.maven.artifact.versioning.ComparableVersion
+case class Version(v: String) {
+  def <(rhs: Version): Boolean = {
+    new ComparableVersion(v).compareTo(new ComparableVersion(rhs.v)) < 0
+  }
```

Comment:
We shouldn't need this, see my comment above (comparison can be done from the `cli` module, using `coursier.core.Version`).

Reply:
I tried, but adding coursier to BloopRifle dependencies causes some resolution errors, unfortunately

Reply:
For some reason guava is resolved to a different version, so we get:
```
Exception in thread "main" java.lang.NoSuchMethodError: com.google.common.base.Preconditions.checkArgument(ZLjava/lang/String;J)V
	at com.google.common.jimfs.WatchServiceConfiguration$PollingConfig.<init>(WatchServiceConfiguration.java:61)
	at com.google.common.jimfs.WatchServiceConfiguration$PollingConfig.<init>(WatchServiceConfiguration.java:55)
	at com.google.common.jimfs.WatchServiceConfiguration.polling(WatchServiceConfiguration.java:44)
	at com.google.common.jimfs.WatchServiceConfiguration.<clinit>(WatchServiceConfiguration.java:35)
	at com.google.common.jimfs.Configuration$Builder.<init>(Configuration.java:346)
	at com.google.common.jimfs.Configuration$Builder.<init>(Configuration.java:318)
	at com.google.common.jimfs.Configuration.builder(Configuration.java:218)
	at com.google.common.jimfs.Configuration$UnixHolder.<clinit>(Configuration.java:90)
	at com.google.common.jimfs.Configuration.unix(Configuration.java:85)
	at com.google.common.jimfs.Configuration.forCurrentPlatform(Configuration.java:212)
	at com.google.common.jimfs.Jimfs.newFileSystem(Jimfs.java:114)
	at com.google.common.jimfs.Jimfs.newFileSystem(Jimfs.java:97)
	at org.scalajs.jsenv.nodejs.ComRun$.setupFile(ComSupport.scala:247)
	at org.scalajs.jsenv.nodejs.ComRun$.start(ComSupport.scala:214)
	at org.scalajs.jsenv.nodejs.NodeJSEnv.startWithCom(NodeJSEnv.scala:46)
	at org.scalajs.testing.adapter.JSEnvRPC.<init>(JSEnvRPC.scala:25)
	at org.scalajs.testing.adapter.TestAdapter.startManagedRunner(TestAdapter.scala:129)
	at org.scalajs.testing.adapter.TestAdapter.$anonfun$getRunnerForThread$1(TestAdapter.scala:120)
	at scala.collection.concurrent.TrieMap.getOrElseUpdate(TrieMap.scala:895)
	at org.scalajs.testing.adapter.TestAdapter.getRunnerForThread(TestAdapter.scala:120)
	at org.scalajs.testing.adapter.TestAdapter.loadFrameworks(TestAdapter.scala:56)
	at scala.build.internal.Runner$.testJs(Runner.scala:247)
	at scala.cli.commands.Test$.$anonfun$testOnce$1(Test.scala:81)
	at scala.cli.commands.Test$.$anonfun$testOnce$1$adapted(Test.scala:76)
	at scala.cli.commands.Run$.withLinkedJs(Run.scala:184)
	at scala.cli.commands.Test$.testOnce(Test.scala:76)
	at scala.cli.commands.Test$.maybeTest$1(Test.scala:30)
	at scala.cli.commands.Test$.run(Test.scala:57)
	at scala.cli.commands.Test$.run(Test.scala:9)
	at caseapp.core.app.CaseApp.main(CaseApp.scala:150)
	at caseapp.core.app.CommandsEntryPoint.main(CommandsEntryPoint.scala:112)
	at scala.cli.ScalaCli$.main(ScalaCli.scala:77)
	at scala.cli.ScalaCli.main(ScalaCli.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at coursier.bootstrap.launcher.a.a(Unknown Source)
	at coursier.bootstrap.launcher.Launcher.main(Unknown Source)
```